### ISSUE TITLE
It’s not obvious you can save settings like vm path etc

### ISSUE THREAD

**Issue by macta** [pharo-project/pharo-launcher#136]
New users don’t easily understand that when you make settings modifications (like vm and image paths or theme) that there is a separate save settings button that will preserve those settings across new version of Launcher.

We’ve discussed a wizard/help screen on new startup, or maybe a separate menu option that calls the existing Pharo functionality ?

**Comment by stephensmith9** [comment]
Also, I think the button should say "Save Settings" instead of "Store Settings". To me, the word "store" implies that I will be asked where to store, more so than the word "save". And "save" is just way more commonly used in similar UI functions.

Even better would be to have an implicit Save. If I have changed a setting, I have already implied that I want it saved, no? Modern UX practice is to save automatically - when I close the settings dialog window it should just save all my changes.

**Comment by stephensmith9** [comment]
And what is the function of the Load Settings button? Since it doesn't give the option to browse to a stored configuration file it might be better named Restore Settings? (with a popup message like 'Restore settings from last saved configuration')

**Comment by demarey** [comment]
@stephensmith9 Right! These buttons are part of the settings framework in Pharo. It could be nice to open an issue in Pharo tracker: https://github.com/pharo-project/pharo/issues
